### ISSUE TITLE
Lab 09 Self Assessment - Jacobs

### ISSUE THREAD

**Issue by jacolb22** [loganbradleyjacobs/lab-09#1]
I learned how to describe features of the dataset I would use, which is an extension of the last lab, but was still somewhat difficult for me, although it was easier now that I had practice. The modeling process is difficult, and is still somewhat unclear to me, even through reading the recipe. However, learning about which model to use, I believe in my case, linear regression, is useful to me, especially for the final project. I will likely take to youtube to find more information regarding this topic, and how to implement it in different ways, probably including outside of this class.

@francojc

**Comment by francojc** [comment]
Yes, the purpose of predictive analysis that we saw was really a type of extension of exploratory analysis --with the caveat that we have an outcome variable which we are interested in predicting. 

The workflow diverges too as we split the dataset into training and testing and then build our model, test, and evaluate it on the training set before finalizing a model and evaluating it on the testing set to give us a much better assessment of how good the model generalizes to new data.

With these ideas in mind, the details will fall into place the more you work with it --whether that be in language, sound, vision, etc.
